### ISSUE TITLE
mobile: Crash on Android

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/ebiten#948]
Device: ONE E1005

```
runtime error: invalid memory address or nil pointer dereference

goroutine 7 [running]:
runtime/debug.Stack(0x4f434e0c, 0x9f4596a0, 0x9f747698)
        /usr/local/go/src/runtime/debug/stack.go:24 +0x80
github.com/hajimehoshi/ebiten/internal/uidriver/mobile.(*UserInterface).run.func1(0x4f434fac)
        /Users/daigosato/Development/Go/src/github.com/hajimehoshi/ebiten/internal/uidriver/mobile/ui.go:193 +0x3c
panic(0x9f4596a0, 0x9f747698)
        /usr/local/go/src/runtime/panic.go:522 +0x1ac
github.com/hajimehoshi/ebiten/internal/shareable.resolveDeferred()
        /Users/daigosato/Development/Go/src/github.com/hajimehoshi/ebiten/internal/shareable/shareable.go:61 +0x3c
github.com/hajimehoshi/ebiten/internal/shareable.init.0.func1(0x0, 0x0)
        /Users/daigosato/Development/Go/src/github.com/hajimehoshi/ebiten/internal/shareable/shareable.go:54 +0x60
github.com/hajimehoshi/ebiten/internal/hooks.RunBeforeUpdateHooks(0x0, 0x0)
        /Users/daigosato/Development/Go/src/github.com/hajimehoshi/ebiten/internal/hooks/hooks.go:38 +0xac
github.com/hajimehoshi/ebiten.(*uiContext).Update(0x4f4901e0, 0x57c58130, 0x2, 0x0)
        /Users/daigosato/Development/Go/src/github.com/hajimehoshi/ebiten/uicontext.go:92 +0x134
github.com/hajimehoshi/ebiten/internal/uidriver/mobile.(*UserInterface).update(0x9f8236a0, 0x9f4baa70, 0x4f4901e0, 0x0, 0x0)
        /Users/daigosato/Development/Go/src/github.com/hajimehoshi/ebiten/internal/uidriver/mobile/ui.go:277 +0x170
github.com/hajimehoshi/ebiten/internal/uidriver/mobile.(*UserInterface).run(0x9f8236a0, 0x1e0, 0x301, 0x0, 0x3fe80000, 0x0, 0x0, 0x9f4baa70, 0x4f4901e0, 0x9f4bc6d0, ...)
        /Users/daigosato/Development/Go/src/github.com/hajimehoshi/ebiten/internal/uidriver/mobile/ui.go:219 +0x184
github.com/hajimehoshi/ebiten/internal/uidriver/mobile.(*UserInterface).RunWithoutMainLoop.func1(0x4f44a140, 0x9f8236a0, 0x1e0, 0x301, 0x0, 0x3fe80000, 0x0, 0x0, 0x9f4baa70, 0x4f4901e0, ...)
        /Users/daigosato/Development/Go/src/github.com/hajimehoshi/ebiten/internal/uidriver/mobile/ui.go:179 +0x94
created by github.com/hajimehoshi/ebiten/internal/uidriver/mobile.(*UserInterface).RunWithoutMainLoop
        /Users/daigosato/Development/Go/src/github.com/hajimehoshi/ebiten/internal/uidriver/mobile/ui.go:177 +0x98
```

**Comment by hajimehoshi** [comment]
CC @wasedaigo

**Comment by hajimehoshi** [comment]
This happens on v1.10.0-alpha.3. Not sure this happens on alpha.4.

**Comment by hajimehoshi** [comment]
I think this should be fixed as of v1.10.0-alpha.4
